api/python/tests/test_data_transfer.py

```diff
@@ -431,9 +433,15 @@ def test_multipart_copy(self):
         )
 
         with mock.patch('quilt3.data_transfer.s3_transfer_config.max_request_concurrency', 1):
-            data_transfer.copy_file_list([
-                (PhysicalKey.from_url('s3://example1/large_file1.npy'), PhysicalKey.from_url('s3://example2/large_file2.npy'), size),
-            ])
+            stdout = io.StringIO()
+
+            # temporarily redirect stderr to capture warnings (usually errors)
+            with redirect_stderr(stdout), mock.patch('quilt3.data_transfer.DISABLE_TQDM', False):
+                data_transfer.copy_file_list([
+                    (PhysicalKey.from_url('s3://example1/large_file1.npy'), PhysicalKey.from_url('s3://example2/large_file2.npy'), size),
+                ])
+            assert re.search('#+', stdout.getvalue())
+            assert re.search('[1-100]', stdout.getvalue())
```

Comment:
This regex is equivalent to `'[01]'`, probably that's not something you actually want. Maybe we should simply check that the buffer contents is empty?

Reply:
>Maybe we should simply check that the buffer contents is empty?

How do you propose we check that?

Reply:
`assert not stdout.getvalue()`?

Reply:
Anything can be captured by `stderr` don't you think it's better to check for `tqdm` specific outputs?

Reply:
1. if something is written to `stderr` during `copy_file_list()` call this might be the problem that we want to be aware of
2. I don't think that `#` and digits are very specific indicators of `tqdm` output

Reply:
Alright cool